multicluster/controllers/multicluster/core/leader_elector.go

```diff
@@ -123,7 +129,7 @@ func (m *remoteClusterManager) DoLeaderElection() {
 		electedLeaderClusterID := connectedClusterIDs[electedLeaderIndex]
 		m.electedLeaderCluster = m.remoteClusters[electedLeaderClusterID]
 		m.needElection = false
-		klog.Infof("Election completed", "leader", electedLeaderClusterID)
+		klog.InfoS("Election completed ", "leader", electedLeaderClusterID)
```

Comment:
no need to add blankspace for `klog.InfoS`, please remove all tailing blankspaces.

Reply:
btw, you forgot to add sign-off in last commit.

Reply:
Maybe you looked before I added, its there now.

Sure will remove the blank spaces

Reply:
Done